simple/src/hello.c

```diff
@@ -36,25 +42,59 @@ void WDOG_disable (void)
 int main(void)
 {
   int counter = 0;
+  int i=0;
 
-  WDOG_disable();             /* Disable Watchdog in case it is not done in startup code */
-                              /* Enable clocks to peripherals (PORT modules) */
-  PCC-> PCCn[PCC_PORTC_INDEX] = PCC_PCCn_CGC_MASK; /* Enable clock to PORT C */
-  PCC-> PCCn[PCC_PORTD_INDEX] = PCC_PCCn_CGC_MASK; /* Enable clock to PORT D */
+  WDOG_disable();             /* DisaLED_BLUEe Watchdog in case it is not done in startup code */
+                              /* EnaLED_BLUEe clocks to peripherals (PORT modules) */
+  PCC-> PCCn[PCC_PORTC_INDEX] = PCC_PCCn_CGC_MASK; /* EnaLED_BLUEe clock to PORT C */
+  PCC-> PCCn[PCC_PORTD_INDEX] = PCC_PCCn_CGC_MASK; /* EnaLED_BLUEe clock to PORT D */
                                /* Configure port C12 as GPIO input (BTN 0 [SW2] on EVB) */
-  PTC->PDDR &= ~(1<<PTC12);    /* Port C12: Data Direction= input (default) */
-  PORTC->PCR[12] = 0x00000110; /* Port C12: MUX = GPIO, input filter enabled */
+  GPIOC->PDDR &= ~(1<<BUTTON1);    /* Port C12: Data Direction= input (default) */
+  PORTC->PCR[BUTTON1] = (eAF_pinGPIO<<PCR_MUX) | (1<<PCR_PFE); /* Port C12: MUX = GPIO, input filter enaLED_BLUEed */
+  	  	  	  	  	  	  	   /* Configure port C13 as GPIO input (BTN 1 [SW3] on EVB) */
+  GPIOC->PDDR &= ~(1<<BUTTON2);    /* Port C13: Data Direction= input (default) */
+  PORTC->PCR[BUTTON2] = (eAF_pinGPIO<<PCR_MUX) | (1<<PCR_PFE); /* Port C13: MUX = GPIO, input filter enaLED_BLUEed */
                                /* Configure port D0 as GPIO output (LED on EVB) */
-  PTD->PDDR |= 1<<PTD0;        /* Port D0: Data Direction= output */
-  PORTD->PCR[0] = 0x00000100;  /* Port D0: MUX = GPIO */
+
+
+  GPIOD->PDDR |= 1<<LED_BLUE;        /* Port D0: Data Direction= output */
+  PORTD->PCR[LED_BLUE] = (eAF_pinGPIO<<PCR_MUX);  /* Port D0: MUX = GPIO */
+
+  GPIOD->PDDR |= 1<<LED_RED;		   /* Port D15: Data Direction= output */
+  PORTD->PCR[LED_RED] = (eAF_pinGPIO<<PCR_MUX);
+
+  GPIOD->PDDR |= 1<<LED_GREEN;		   /* Port D15: Data Direction= output */
+  PORTD->PCR[LED_GREEN] = (eAF_pinGPIO<<PCR_MUX);
 
   for(;;) {
-    if (PTC->PDIR & (1<<PTC12)) {   /* If Pad Data Input = 1 (BTN0 [SW2] pushed) */
-      PTD-> PCOR |= 1<<PTD0;        /* Clear Output on port D0 (LED on) */
+    if (GPIOC->PDIR & (1<<BUTTON2)) {   /* If Pad Data Input = 1 (BTN0 [SW2] pushed) */
+    	counter++;       /* Clear Output on port D0 (LED on) */
+    }
+    if(counter >15){
```

Comment:
thanks, interesting application loop. Remind me to have a 5min discussion about state machines!